App/Services/MutagenController.cs

```diff
@@ -132,11 +122,10 @@
             _sessionCount += 1;
         }
 
-        return session;
+        throw new NotImplementedException();
```

Comment:
This breaks the UTs --- and doesn't seem to be related to the UI view. Did it get added by accident to this PR?

Reply:
My bad, with the type changes I made I had to change the output of this function and wasn't thinking about UTs lol.